svf/include/Util/SVFBugReport.h

```diff
@@ -303,21 +294,71 @@ class SVFBugReport
      *      it will add the bug into bugQueue.
      * usage: addBug<GenericBug::BOA>(BufferOverflowBug(bugInst, 0, 10, 1, 11))
      */
-    template <typename T>
-    void addBug(T bug)
+    void addSaberBug(GenericBug::BugType bugType, const GenericBug::EventStack &eventStack)
     {
-        T *newBug = new T(bug);
-        bugSet.insert(newBug);
+        GenericBug *newBug;
+        switch(bugType){
+        case GenericBug::NEVERFREE:{
+            newBug = new NeverFreeBug(eventStack);
+            bugSet.insert(newBug);
```

Comment:
bugSet.insert(new NeverFreeBug(eventStack));

Reply:
newBug is use at line 333 `newBug->printBugToTerminal()`